plugins/push/frontend/public/javascripts/countly.models.js

```diff
@@ -1,8 +1,8 @@
 /*global countlyVue, CV, countlyCommon, Promise, moment*/
 (function(countlyPushNotification) {
 
-    var messagesSentLabel = CV.i18n('push-notification.time-chart-serie-messages-sent');
-    var actionsPerformedLabel = CV.i18n('push-notification.time-chart-serie-actions-performed');
+    var messagesSentLabel = CV.i18n('push-notification.messages-sent-serie-name');
+    var actionsPerformedLabel = CV.i18n('push-notification.actions-performed-serie-name');
```

Comment:
Different i18n keys. By mistake or on purpose?

Reply:
On purpose,just trying to make more sense I guess with my local.